qemu/tests/cfg/balloon_memhp.cfg

```diff
@@ -0,0 +1,41 @@
+# Notes:
+#   Need add movable_node in kernel line for rhel8 guest, rhel7 won't work well
+#   with memory unplug.
+- balloon_memhp:
+    type = balloon_memhp
+    slots_mem = 4
+    size_mem = 1G
+    maxmem_mem = 32G
+    login_timeout = 600
+    no Host_RHEL.6
+    no RHEL.5
+    no Windows..i386
+    no WinXP Win2000 Win2003 WinVista
+    guest_numa_nodes = "node0 node1"
+    target_mem = "plug"
+    balloon = balloon0
+    balloon_dev_devid = balloon0
+    balloon_dev_add_bus = yes
+    Windows:
+        driver_name = "balloon"
+        guest_compare_threshold = 300
+        cdroms = "cd1 winutils virtio"
```

Comment:
Seems cd1 and winutils are dup. Remove winutils works enough.
winuils should be added in default, we can use cdroms += " virtio"